src/target/stm32l4.c

```diff
@@ -168,6 +168,8 @@ typedef enum stm32l4_device_id {
 	 */
 	ID_STM32WLXX = 0x4970U, /* from RM0461, Rev.5 §36.4.5, and RM0453, Rev.3 §38.4.5 */
 	ID_STM32WBXX = 0x4950U, /* from RM0434, Rev.10 §41.4.8 */
+	ID_STM32WB1X = 0x4940u, /* from RM0473, Rev.7 §33.4.8 and RM0478 Rev.5 §31.4.8 */
```

Comment:
The 'U' on the end of this needs to be upper case (clang-tidy lint warning), and we generally don't leave a blank line at the end of an enum like this